cmd/neofs-node/config.go

```diff
@@ -830,6 +846,8 @@ func (c *cfg) needBootstrap() bool {
 // It is calculated as size/capacity ratio of "remote object put" worker.
 // Returns float value between 0.0 and 1.0.
 func (c *cfg) ObjectServiceLoad() float64 {
+	c.cfgObject.poolLock.RLock()
+	defer c.cfgObject.poolLock.RUnlock()
```

Comment:
You can avoid the lock by replacing `putRemoteCapacity` with https://pkg.go.dev/github.com/panjf2000/ants/v2#Pool.Cap